OpenRA.Mods.Example/Traits/Player/OverlayPlayerColorPalette.cs

```diff
@@ -0,0 +1,52 @@
+﻿using OpenRA.Graphics;
+using OpenRA.Primitives;
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.Example.Traits.Player
+{
+	[Desc("More beautiful variant of the PlayerColorPalette by using the overlay blend mode.")]
```

Comment:
I don't see this using an (OpenGL) blend mode. This is a bit confusing to me.

Reply:
OpenGL didn't invent blend modes - it just implements standard algorithms, like the one this trait uses ;)